core/gethwrappers/ccip/generated/latest/message_hasher/message_hasher.go

```diff
@@ -32,7 +32,7 @@ type ClientEVMExtraArgsV1 struct {
 	GasLimit *big.Int
 }
 
-type ClientEVMExtraArgsV2 struct {
```

Comment:
The old name ClientEVMExtraArgsV2 is still referenced in some tests inside the ccip.ccipevm package.
They too need to change to this new name.

Reply:
Fixed! Forgot we have to copy the new wrapper to the 1.6 directory 👍🏼